src/main/java/wolox/training/exceptions/BookNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package wolox.training.exceptions;
+
+public class BookNotFoundException extends RuntimeException{
```

Comment:
Lo recomendable para evitar dejar los mensajes de errores en el controller, es que se agregue la anotación @ResponseStatus(code = HttpStatus.NOT_FOUND, reason = "Book not found") en la clase junto con el code y reason de la exception.